bin/generate-tls.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# Script can be used to either automatically generate a self-signed TLS certificate, or to provide your own TLS cert and key.
```

Comment:
Suggestion: This script creates a secret called wildcard.<namespace>.<domain>, which is inherited by the ingress and can be used by all FR components. You can either provide your own TLS cert and key, or you can have the script generate a self-signed TLS certificate.

Reply:
Ah yes, this is slightly confusing.  You can actually use the script to upload your own cert and keys too.  You just need to provide the path to the keys.  I'll reword it so its clearer.  So the script can be used in both use cases

Reply:
I've made it clearer that the script does both